Source/Ultraviolet.FMOD/Shared/Native/FMODNative.cs

```diff
@@ -61,6 +61,9 @@ static FMODNative()
         
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static FMOD_RESULT FMOD_System_Update(FMOD_SYSTEM* system) => impl.FMOD_System_Update(system);
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static FMOD_RESULT FMOD_System_SetCallback(FMOD_SYSTEM* system, FMOD_SYSTEM_CALLBACK callback, FMOD_SYSTEM_CALLBACK_TYPE callbackmask = FMOD_SYSTEM_CALLBACK_TYPE.ALL) => impl.FMOD_System_SetCallback(system, callback, callbackmask);
```

Comment:
It looks like you modified `FMODNative` manually. Changes to the native API surface should be made by updating `_Definitionx.xml` and running the `UvNativeCodeGen` project, which will rebuild these files for you. This helps keep things in sync across the different platform implementations, and will become especially import when .NET 6 (theoretically) unifies all of this stuff, as I'll need to regenerate all of the native interface code to fit with whatever that ends up looking like.

Reply:
Didn't know about that. I've pushed the fix!